kpops/cli/main.py

```diff
@@ -63,13 +63,13 @@
 )
 
 CONFIG_PATH_OPTION: Path = typer.Option(
-    default=Path("config.yaml"),
+    default=Path(),
     exists=False,
```

Comment:
should we enable this check?

Reply:
Sure

Reply:
no, I don't know. Which output do you get with it enabled if the directory doesn't exist?

Reply:
I am fine with leaving it disabled if that improves the user experience because of our custom error, but then we should add a comment here explaining why it's disabled.

Reply:
enabled it and removed the custom check